### ISSUE TITLE
Jax not picking up the GPU in my Linux

### ISSUE THREAD

**Issue by WeheliyeHashi** [kirkegaardlab/deeptangle#6]
Hi 
Thank you for the code! I was wondering why JAX is not picking up my GPU. I have UBUNTu with cuda 11.3 and cudnn 8.2. I am currently training on the CPU and its taking a log time. Any suggestion on how I can pick up the GPu using jax?  Thanks!

**Comment by juliusbierk** [comment]
This is a problem with `jax` and not with `deeptangle`. Please check that you installed `jax` according to instructions at https://github.com/google/jax/.

Also, please note that we include pretrained [weights](https://github.com/kirkegaardlab/deeptangle#weights) and therefore, unless you change the model, there should be no need for you to train it yourself.

**Comment by WeheliyeHashi** [comment]
Thanks @juliusbierk.   You have an excellent code.
I will follow the instructions for installation of jax. I have already tried your weights on a real datset which I collected and it doesn't detect the splines well. I am happy to share the video which I am trying to inference. 
 
I have real C.elegan dataset along with their splines which I want to train on your code and get new weights and see if those work better.
